charmcraft/templates/init/run_tests

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+# Copyright (c) {{ year }} {{ author }}
+# See LICENSE file for licensing details.
+
+flake8 --max-line-length=99
+PYTHONPATH=src${PYTHONPATH:+:}$PYTHONPATH python3 -m unittest -v "$@"
```

Comment:
behold, the magic of Bash !

Reply:
Will this work on windows?

Can't we get away without fixing the PATH? What about something like `env/bin/python -m unittest discover tests/`

Reply:
Also you probably should be able to improve the README saying that just run this file, and here you do:

```
venv/bin/flake8...
venv/bin/python3 -m unittest discover...
```

Reply:
it's not bash but `/bin/sh`, and it'll work on windows as much as charmcraft's and the operator's one will :-)

Reply:
I don't think there's a way for things to work without fixing PYTHONPATH